infra/bootstrap/tenant/config/aad-groups/portalAADGroups.defradev.json

```diff
@@ -0,0 +1,23 @@
+{
+    "userADGroups": [
+        {
+            "displayName": "AAG-Users-ADP-Portal",
+            "description": "Azure AD Access group with Sign in permissions to the ADP Portal.",
+            "Owners": {
+                "serviceprincipals" : [
+                    "ADO-DefraGovUK-AAD-ADP-SSV3"
+                ],
+                "users" : #{{ noescape(portalUserAndAccessAADGroupOwners) }}
+            },
+            "Members": {
+                "groups" : [
+                    "AG-Azure-CDO-ADP-PlatformEngineers",
+                    "GG-Users-ADP-PlatformEngineers",
```

Comment:
what is GG?

Reply:
GG is me 🫣. @santosh-gh : you can remove that group.

Reply:
Removed the GG user group